networking/libsnnet/network.go

```diff
@@ -189,3 +189,14 @@ type GreTunEP struct {
 	CNCIId   string // UUID of the CNCI
 	CNId     string // UUID of the CN
 }
+
+// GreTunEP ciao GRE Tap device representation
```

Comment:
Just a nit about the comment. 
This is actually a GRE point to multi-point tunnel. Not a simple GRE Tunnel.

Reply:
thanks, I will make a more accurate comment.